DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/DelegateGroupsControllerTests.cs

```diff
@@ -455,5 +455,69 @@ public void EditDelegatesGroupDescription_should_redirect_to_index_action()
 
             result.Should().BeRedirectToActionResult().WithActionName("Index");
         }
+
+        [Test]
+        public void EditGroupName_should_redirect_to_index_action_when_update_is_successful()
+        {
+            // Given
+            const int groupId = 103;
+            const int centreId = 2;
+            var model = new EditGroupNameViewModel()
+            {
+                GroupName = "Test Group Name",
+            };
+
+            A.CallTo(() => groupsDataService.UpdateGroupName(
+                groupId,
+                centreId,
+                model.GroupName)).DoesNothing();
+
+            // When
+            var result = delegateGroupsController.EditGroupName(model, groupId);
+
+            // Then
+            A.CallTo(() => groupsDataService.UpdateGroupName(
+                    groupId,
+                    centreId,
+                    model.GroupName))
+                .MustHaveHappened();

```

Comment:
Formatter should probably be putting the final brackets on the next line in both A.CallTo cases here.

Reply:
I have ran the formatter